packages/components/src/Label/Label.module.scss

```diff
@@ -0,0 +1,120 @@
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/typography";
+@import "~@kaizen/design-tokens/sass/color";
+@import "../../styles/form-variables";
+
+$label-start-margin: $spacing-xs;
+
+.label,
+:global(.ideal-sans) .label {
+  // override Murmur global styles :(
+  opacity: inherit;
+  color: $dt-color-form-text-color;
+  visibility: visible;
+  width: 100%;
+  text-align: left; // fallback for IE and Edge
+  text-align: start;
+
+  &[dir="rtl"],
+  [dir="rtl"] & {
+    text-align: right; // fallback for IE and Edge
+    text-align: start;
+  }
+}
+
+.reversed {
+  color: $color-white;
+
+  a {
```

Comment:
I don't know why we override anchor styles here, but I suppose it means it should be added to the stickersheet.